lib/liquid/combinators/tags/custom_block.ex

```diff
@@ -1,33 +1,23 @@
 defmodule Liquid.Combinators.Tags.CustomBlock do
+  @moduledoc """
```

Comment:
This module doc can be more like other modules. Check @moduledoc for other tags.

Reply:
done